src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -703,6 +703,22 @@ def CancelPathfinding() -> None:
     """
     pass
 
+def GetPath(x: int, y: int, z: int = 1337, distance: int = 1) -> Any:
```

Ruff (0.12.2)

Undefined name `Any`

(F821)